services/121-service/test/payment/do-payment-fsp-nedbank.test.ts

```diff
@@ -229,7 +225,7 @@ describe('Do payment', () => {
         const getTransactionsBody = (
           await getTransactions({
             programId,
-            paymentNr: payment,
+            paymentId: doPaymentResponse.body.id,
```

Comment:
I notice you're going back and forth a bit between adding a specific line 'const paymentId = doPaymentResponse.body.id' and not. Doesn't really matter much of course, but if you want to align, I like the former better.